app/views/layouts/listnav/_ems_middleware.html.haml

```diff
@@ -5,8 +5,11 @@
 
     = miq_accordion_panel(_("Properties"), false, "ems_middleware_prop") do
       %ul.nav.nav-pills.nav-stacked
-        %li
-          = link_to(_('Summary'), {:action => 'show', :id => @record, :display => 'main'}, :title => _("Show Summary"))
+        = li_link(:if => true,
```

Comment:
`:if => true`? really? is the helper method designed so bad or is this a bug in this PR?